docs/pages/get-started/expo-go.mdx

```diff
@@ -1,14 +1,19 @@
 ---
 title: Expo Go
-description: Learn about Expo Go, the free, open-source client for developing and testing React Native apps on Android and iOS.
+description: Learn about Expo Go, the free, open-source sandbox app for learning and experimenting with React Native on Android and iOS.
 ---
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { BookOpen02Icon } from '@expo/styleguide-icons';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
 
-[Expo Go](https://expo.dev/client) is a free, [open-source](https://github.com/expo/expo/tree/main/home) client for testing React Native apps on Android and iOS devices without building anything locally. It allows you to open up apps served through Expo CLI and run your projects faster when developing them.
+[Expo Go](https://expo.dev/client) is a free, [open-source](https://github.com/expo/expo/tree/main/home) sandbox for learning and experimenting with React Native on Android and iOS devices. You can install it directly from App Stores, and get up and running within minutes — no need to install a native toolchain and compile an app.
+
+Expo Go is not recommended for building production-quality apps — use [development builds](/develop/development-builds/introduction.mdx) instead. It's also not recommend for deploying/distributing apps, [learn more about deployment](/deploy/build-project/). That said, everything that you learn while working in Expo Go will be transferable to development builds, and you're likely to come back to Expo Go along your journey when you want to try new ideas in a quick new project sandbox. It's a great place to start!
```

Comment:
```suggestion
Expo Go is not recommended for building production-quality apps &mdash; use [development builds](/develop/development-builds/introduction) instead. It's also not recommended for [deploying/distributing apps](/deploy/build-project/). That said, everything you learn while working in Expo Go will be transferable to development builds, and you're likely to come back to Expo Go along your journey when you want to try new ideas in a quick new project sandbox. It's a great place to start!
```

- replaced dash with mdash;
- Fix typo
- I'd suggest we put the link on the context instead of adding the context by "learn more...". It's makes it easier for readers to scan through quickly without grabbing too much of their attention.